Update response-helpers.md

AmrSaber opened the pull request:
Update the mentioned redirection status message.

307 code corresponds to "Temporary redirect" not "Found" (the status for Found is 302) as mentioned in [httpstatuses documentation](https://httpstatuses.com/307).

ijjk commented:
## Failing test suites <!--1504767579-->
Commit: df66916ecd512a31bee87c75c4b1d3ca1ef00360

**test/integration/dist-dir/test/index.test.js** 
- distDir > should handle null/undefined distDir
- distDir > With basic usage > should not build the app within the default `.next` directory

<details>
<summary>Expand output</summary>

  ● distDir › With basic usage › should not build the app within the default `.next` directory

    expect(received).toBeFalsy()

    Received: true

      40 |       expect(
      41 |         await fs.exists(join(__dirname, `/../.next/${BUILD_ID_FILE}`))
    > 42 |       ).toBeFalsy()
         |         ^
      43 |     })
      44 |   })
      45 | 

      at Object.<anonymous> (integration/dist-dir/test/index.test.js:42:9)

  ● distDir › should handle null/undefined distDir

    expect(received).toBe(expected) // Object.is equality

    Expected: 0
    Received: 189

      61 |     await fs.writeFile(nextConfig, origNextConfig)
      62 | 
    > 63 |     expect(stderr.length).toBe(0)
         |                           ^
      64 |   })
      65 | })
      66 | 

      at Object.<anonymous> (integration/dist-dir/test/index.test.js:63:27)

</details>

ijjk commented:
# Stats from current PR

<details>
<summary><strong>Default Server Mode</strong> (Decrease detected ✓)</summary>

<br/>

<details>
<summary><strong>General</strong></summary>

|  | vercel/next.js canary  | AmrSaber/next.js patch-1 | Change |
| - | - | - | - |
| buildDuration | 12.4s | 12.2s | -217ms |
| nodeModulesSize | 85 MB | 85 MB | ✓ |

</details>

<details>
<summary><strong>Page Load Tests</strong> Overall decrease ⚠️</summary>

|  | vercel/next.js canary  | AmrSaber/next.js patch-1 | Change |
| - | - | - | - |
| / failed reqs | 0 | 0 | ✓ |
| / total time (seconds) | 2.258 | 2.349 | ⚠️ +0.09 |
| / avg req/sec | 1107.26 | 1064.26 | ⚠️ -43 |
| /error-in-render failed reqs | 0 | 0 | ✓ |
| /error-in-render total time (seconds) | 1.239 | 1.228 | -0.01 |
| /error-in-render avg req/sec | 2018.53 | 2035.63 | +17.1 |

</details>

<details>
<summary><strong>Client Bundles (main, webpack, commons)</strong></summary>

|  | vercel/next.js canary  | AmrSaber/next.js patch-1 | Change |
| - | - | - | - |
| 677f882d2ed8..d9fb.js gzip | 12.7 kB | 12.7 kB | ✓ |
| framework.HASH.js gzip | 39 kB | 39 kB | ✓ |
| main-6b336ff..ab65.js gzip | 6.53 kB | 6.53 kB | ✓ |
| webpack-e067..f178.js gzip | 751 B | 751 B | ✓ |
| Overall change | 58.9 kB | 58.9 kB | ✓ |

</details>

<details>
<summary><strong>Client Bundles (main, webpack, commons) Modern</strong></summary>

|  | vercel/next.js canary  | AmrSaber/next.js patch-1 | Change |
| - | - | - | - |
| 677f882d2ed8..dule.js gzip | 8.23 kB | 8.23 kB | ✓ |
| framework.HA..dule.js gzip | 39 kB | 39 kB | ✓ |
| main-878859a..dule.js gzip | 5.64 kB | 5.64 kB | ✓ |
| webpack-07c5..dule.js gzip | 751 B | 751 B | ✓ |
| Overall change | 53.6 kB | 53.6 kB | ✓ |

</details>

<details>
<summary><strong>Legacy Client Bundles (polyfills)</strong></summary>

|  | vercel/next.js canary  | AmrSaber/next.js patch-1 | Change |
| - | - | - | - |
| polyfills-4b..e242.js gzip | 31 kB | 31 kB | ✓ |
| Overall change | 31 kB | 31 kB | ✓ |

</details>

<details>
<summary><strong>Client Pages</strong></summary>

|  | vercel/next.js canary  | AmrSaber/next.js patch-1 | Change |
| - | - | - | - |
| _app-3b0cf13..85f8.js gzip | 1.28 kB | 1.28 kB | ✓ |
| _error-6f635..c393.js gzip | 3.44 kB | 3.44 kB | ✓ |
| hooks-d4ffc3..9e0f.js gzip | 887 B | 887 B | ✓ |
| index-17468f..5d83.js gzip | 227 B | 227 B | ✓ |
| link-d9a15a0..130a.js gzip | 1.57 kB | 1.57 kB | ✓ |
| routerDirect..924c.js gzip | 284 B | 284 B | ✓ |
| withRouter-7..c13d.js gzip | 284 B | 284 B | ✓ |
| Overall change | 7.98 kB | 7.98 kB | ✓ |

</details>

<details>
<summary><strong>Client Pages Modern</strong></summary>

|  | vercel/next.js canary  | AmrSaber/next.js patch-1 | Change |
| - | - | - | - |
| _app-75d3a82..dule.js gzip | 625 B | 625 B | ✓ |
| _error-f2fcc..dule.js gzip | 2.29 kB | 2.29 kB | ✓ |
| hooks-cbf13f..dule.js gzip | 387 B | 387 B | ✓ |
| index-b9a643..dule.js gzip | 226 B | 226 B | ✓ |
| link-973cce3..dule.js gzip | 1.51 kB | 1.51 kB | ✓ |
| routerDirect..dule.js gzip | 284 B | 284 B | ✓ |
| withRouter-f..dule.js gzip | 282 B | 282 B | ✓ |
| Overall change | 5.6 kB | 5.6 kB | ✓ |

</details>

<details>
<summary><strong>Client Build Manifests</strong></summary>

|  | vercel/next.js canary  | AmrSaber/next.js patch-1 | Change |
| - | - | - | - |
| _buildManifest.js gzip | 321 B | 321 B | ✓ |
| _buildManife..dule.js gzip | 330 B | 330 B | ✓ |
| Overall change | 651 B | 651 B | ✓ |

</details>

<details>
<summary><strong>Rendered Page Sizes</strong></summary>

|  | vercel/next.js canary  | AmrSaber/next.js patch-1 | Change |
| - | - | - | - |
| index.html gzip | 968 B | 968 B | ✓ |
| link.html gzip | 975 B | 975 B | ✓ |
| withRouter.html gzip | 962 B | 962 B | ✓ |
| Overall change | 2.9 kB | 2.9 kB | ✓ |

</details>

</details>
<hr/>
<details>
<summary><strong>Serverless Mode</strong></summary>

<br/>

<details>
<summary><strong>General</strong></summary>

|  | vercel/next.js canary  | AmrSaber/next.js patch-1 | Change |
| - | - | - | - |
| buildDuration | 13.8s | 13.5s | -310ms |
| nodeModulesSize | 85 MB | 85 MB | ✓ |

</details>

<details>
<summary><strong>Client Bundles (main, webpack, commons)</strong></summary>

|  | vercel/next.js canary  | AmrSaber/next.js patch-1 | Change |
| - | - | - | - |
| 677f882d2ed8..d9fb.js gzip | 12.7 kB | 12.7 kB | ✓ |
| framework.HASH.js gzip | 39 kB | 39 kB | ✓ |
| main-6b336ff..ab65.js gzip | 6.53 kB | 6.53 kB | ✓ |
| webpack-e067..f178.js gzip | 751 B | 751 B | ✓ |
| Overall change | 58.9 kB | 58.9 kB | ✓ |

</details>

<details>
<summary><strong>Client Bundles (main, webpack, commons) Modern</strong></summary>

|  | vercel/next.js canary  | AmrSaber/next.js patch-1 | Change |
| - | - | - | - |
| 677f882d2ed8..dule.js gzip | 8.23 kB | 8.23 kB | ✓ |
| framework.HA..dule.js gzip | 39 kB | 39 kB | ✓ |
| main-878859a..dule.js gzip | 5.64 kB | 5.64 kB | ✓ |
| webpack-07c5..dule.js gzip | 751 B | 751 B | ✓ |
| Overall change | 53.6 kB | 53.6 kB | ✓ |

</details>

<details>
<summary><strong>Legacy Client Bundles (polyfills)</strong></summary>

|  | vercel/next.js canary  | AmrSaber/next.js patch-1 | Change |
| - | - | - | - |
| polyfills-4b..e242.js gzip | 31 kB | 31 kB | ✓ |
| Overall change | 31 kB | 31 kB | ✓ |

</details>

<details>
<summary><strong>Client Pages</strong></summary>

|  | vercel/next.js canary  | AmrSaber/next.js patch-1 | Change |
| - | - | - | - |
| _app-3b0cf13..85f8.js gzip | 1.28 kB | 1.28 kB | ✓ |
| _error-6f635..c393.js gzip | 3.44 kB | 3.44 kB | ✓ |
| hooks-d4ffc3..9e0f.js gzip | 887 B | 887 B | ✓ |
| index-17468f..5d83.js gzip | 227 B | 227 B | ✓ |
| link-d9a15a0..130a.js gzip | 1.57 kB | 1.57 kB | ✓ |
| routerDirect..924c.js gzip | 284 B | 284 B | ✓ |
| withRouter-7..c13d.js gzip | 284 B | 284 B | ✓ |
| Overall change | 7.98 kB | 7.98 kB | ✓ |

</details>

<details>
<summary><strong>Client Pages Modern</strong></summary>

|  | vercel/next.js canary  | AmrSaber/next.js patch-1 | Change |
| - | - | - | - |
| _app-75d3a82..dule.js gzip | 625 B | 625 B | ✓ |
| _error-f2fcc..dule.js gzip | 2.29 kB | 2.29 kB | ✓ |
| hooks-cbf13f..dule.js gzip | 387 B | 387 B | ✓ |
| index-b9a643..dule.js gzip | 226 B | 226 B | ✓ |
| link-973cce3..dule.js gzip | 1.51 kB | 1.51 kB | ✓ |
| routerDirect..dule.js gzip | 284 B | 284 B | ✓ |
| withRouter-f..dule.js gzip | 282 B | 282 B | ✓ |
| Overall change | 5.6 kB | 5.6 kB | ✓ |

</details>

<details>
<summary><strong>Client Build Manifests</strong></summary>

|  | vercel/next.js canary  | AmrSaber/next.js patch-1 | Change |
| - | - | - | - |
| _buildManifest.js gzip | 321 B | 321 B | ✓ |
| _buildManife..dule.js gzip | 330 B | 330 B | ✓ |
| Overall change | 651 B | 651 B | ✓ |

</details>

<details>
<summary><strong>Serverless bundles</strong></summary>

|  | vercel/next.js canary  | AmrSaber/next.js patch-1 | Change |
| - | - | - | - |
| _error.js | 918 kB | 918 kB | ✓ |
| 404.html | 4.61 kB | 4.61 kB | ✓ |
| hooks.html | 3.86 kB | 3.86 kB | ✓ |
| index.js | 918 kB | 918 kB | ✓ |
| link.js | 976 kB | 976 kB | ✓ |
| routerDirect.js | 969 kB | 969 kB | ✓ |
| withRouter.js | 969 kB | 969 kB | ✓ |
| Overall change | 4.76 MB | 4.76 MB | ✓ |

</details>

</details>
Commit: df66916ecd512a31bee87c75c4b1d3ca1ef00360

AmrSaber commented:
Whatever tests failing, I just removed one word in the docs and replaced it with 3 other words. So surely my updates did not cause these tests to fail.

lfades reviewed (approved):
Thank you @AmrSaber 💯
